lib/tasks/seed_training.rake

```diff
@@ -0,0 +1,20 @@
+#Let's do this: rake db:seed:training should create 20 accounts as follows:
+
+#email: training-account-#{i}@test.com
+#password: P4ssword
+#password_confirmation: P4ssword
+#name: Training Account #{i}
+
+namespace :db do
+  namespace :seed do
+    task :training => :environment do
```

Comment:
So that this shows up when we run `rake -T` (the rails command for seeing available rake tasks), we'll want to add a `desc`. For an example of this, see https://github.com/colinxfleming/dcaf_case_management/blob/master/lib/tasks/user.rake#L2 , and see how the string after `desc` shows up next to the task when you run `rake -T`.